starfish/starfish.py

```diff
@@ -7,10 +7,13 @@
 import sys
 from pstats import Stats
 
+import numpy as np
```

Comment:
I would discourage you from importing numpy at the top level.  Rationale: it makes starfish --help really really slow.